docs/documentation/server_admin/topics/authentication/conditions.adoc

```diff
@@ -24,7 +24,7 @@ This checks if the other executions in the flow are configured for the user.
 The Execution requirements section includes an example of the OTP form.
 
 `Condition - User Attribute`::
-This checks if the user has set up the required attribute.
+This checks if the user or any of the joined groups have set up the required attribute.
```

Comment:
@pedroigor Maybe we can add additional option to this authenticator like `Allow group attributes` - When false (the default), the behaviour would be the same like today (so not checking groups). When true, it can also check attributes from groups (the tooltip should have the info about the fact that using group attributes can have some performance impact). WDYT?